ZhaoYuLin/Dog.java

```diff
@@ -6,19 +6,50 @@
     int age;
     boolean rabies;
 
-    public Dog(String name, String type, int age) {
+    public Dog(String name,String type,int age) {
         this.name = name;
         breed = type;
         this.age = age;
         rabies = false;
     }
-   
+
     public void bark() {
         System.out.println("bork");
     }
-    
+
     public static void main(String[] args) {
         Dog foo = new Dog("foo", "golden retriever", 5);
         foo.bark();
+        Dog Fred = new Dog();
+        Fred.bark();
+        System.out.println("Fred's age: " + Fred.getAge());
+        Fred.setRabies(true);
+          Dog bar = new Dog("bar", "dachshund",1);
+        System.out.println("Bar's age in dog years: " + bar.getAgeinDogyears());
+        bar.setRabies(true);
+        System.out.println("Does bar have rabies: " + bar.hasRabies());
+
+
+    }
+    public Dog(){
```

Comment:
In general, your constructors should go together near the top. Put this constructor right above the other one.